packages/snap/src/keyring.ts

```diff
@@ -192,14 +193,33 @@ export class SimpleKeyring implements Keyring {
     await this.#saveState();
   }
 
+  #getCurrentUrl(): string {
+    const dappUrlPrefix =
+      process.env.NODE_ENV === 'production'
```

Comment:
Shouldn't it be?

```ts
      process.env.NODE_ENV === 'development'
        ? process.env.DAPP_ORIGIN_DEVELOPMENT
        : process.env.DAPP_ORIGIN_PRODUCTION;
```

I'm asking because I don't see the enviroment being set to `'production'` in the configs.

Reply:
@danroc the [default value](https://github.com/MetaMask/snaps/tree/main/packages/snaps-cli#environment) is production. even when developing locally it was set to production so I had to add `"start": "NODE_ENV='development' mm-snap watch"` to the yarn start command to get it to work with development.

Reply:
Understood. We might want to switch the logic to not rely on the default value.

Reply:
we added the default value check in the snaps config and tested it with and without and the result was the same. I tend to agree though, it would probably be best to be explicit about it.